compiler/src/latex_printer.ml

```diff
@@ -36,6 +45,24 @@ let pp_opt p fmt =
 let pp_paren p fmt =
   F.fprintf fmt "(%a)" p
 
+let pp_string fmt s =
+  F.asprintf "%S" s |>
+  String.iter @@ function
+  | '\\' -> backslash fmt ()
+  | '\'' -> quotesingle fmt ()
+  | '"' -> quotedouble fmt ()
+  | '#' -> sharp fmt ()
+  | '{' -> openbrace fmt ()
+  | '}' -> closebrace fmt ()
+  | '%' -> percent fmt ()
+  | '$' -> dollar fmt ()
+  | '_' -> underscore fmt ()
```

Comment:
Escaping underscores is not necessary. I trust you that you’ve check it does not harm.